### ISSUE TITLE
Git error while installing cypress-cli

### ISSUE THREAD

**Issue by shauvik** [cypress-io/cypress#124]
cypress-cli needs git to be installed for fetching node-xvfb but after installing git, i get this error. 
IMO it should just fetch the dependency from github/release instead of checking it out.

```
# npm install -g cypress-cli
npm WARN addRemoteGit Error: Command failed: git config --get remote.origin.url
npm WARN addRemoteGit 
npm WARN addRemoteGit     at ChildProcess.exithandler (child_process.js:213:12)
npm WARN addRemoteGit     at emitTwo (events.js:87:13)
npm WARN addRemoteGit     at ChildProcess.emit (events.js:172:7)
npm WARN addRemoteGit     at maybeClose (internal/child_process.js:827:16)
npm WARN addRemoteGit     at Process.ChildProcess._handle.onexit (internal/child_process.js:211:5)
npm WARN addRemoteGit  cypress-io/node-xvfb resetting remote /root/.npm/_git-remotes/git-github-com-cypress-io-node-xvfb-git-7b167375 because of error: { [Error: Command failed: git config --get remote.origin.url
npm WARN addRemoteGit ]
npm WARN addRemoteGit   killed: false,
npm WARN addRemoteGit   code: 1,
npm WARN addRemoteGit   signal: null,
npm WARN addRemoteGit   cmd: 'git config --get remote.origin.url' }
/usr/bin/cypress -> /usr/lib/node_modules/cypress-cli/bin/cypress
cypress-cli@0.11.0 /usr/lib/node_modules/cypress-cli
├── xvfb@0.3.0
├── human-interval@0.1.5
├── progress@1.1.8
├── home-or-tmp@2.0.0 (os-homedir@1.0.1, os-tmpdir@1.0.1)
├── commander@2.9.0 (graceful-readlink@1.0.1)
├── coffee-script@1.10.0
├── request-progress@0.3.1 (throttleit@0.0.2)
├── chalk@1.1.3 (supports-color@2.0.0, escape-string-regexp@1.0.5, ansi-styles@2.2.1, has-ansi@2.0.0, strip-ansi@3.0.1)
├── bluebird@3.3.4
├── yauzl@2.4.1 (fd-slicer@1.0.1)
├── through2@2.0.1 (xtend@4.0.1, readable-stream@2.0.6)
├── extract-zip@1.5.0 (debug@0.7.4, mkdirp@0.5.0, concat-stream@1.5.0)
├── fs-extra@0.22.1 (graceful-fs@4.1.3, jsonfile@2.3.0, rimraf@2.5.2)
├── request@2.72.0 (aws-sign2@0.6.0, forever-agent@0.6.1, tunnel-agent@0.4.2, caseless@0.11.0, oauth-sign@0.8.1, is-typedarray@1.0.0, stringstream@0.0.5, isstream@0.1.2, json-stringify-safe@5.0.1, extend@3.0.0, tough-cookie@2.2.2, node-uuid@1.4.7, qs@6.1.0, combined-stream@1.0.5, mime-types@2.1.10, form-data@1.0.0-rc4, bl@1.1.2, aws4@1.3.2, hawk@3.1.3, http-signature@1.1.1, har-validator@2.0.6)
├── update-notifier@0.5.0 (is-npm@1.0.0, string-length@1.0.1, semver-diff@2.1.0, repeating@1.1.3, configstore@1.4.0, latest-version@1.0.1)
└── lodash@3.10.1
```

**Comment by RandallKent** [comment]
The code for this is done, but this has yet to be released. We'll update the issue and reference the changelog when it's released.

**Comment by brian-mann** [comment]
Fixed in [`0.20.0`](https://docs.cypress.io/guides/references/changelog.html#0-20-0)
